Documentation/deploy-master.md

```diff
@@ -369,12 +499,38 @@ Now we can create the `kube-system` namespace:
 $ curl -H "Content-Type: application/json" -XPOST -d'{"apiVersion":"v1","kind":"Namespace","metadata":{"name":"kube-system"}}' "http://127.0.0.1:8080/api/v1/namespaces"
 ```
 
+The Calico policy-agent runs in its own `calico-system` namespace:
```

Comment:
> The Calico policy-agent runs in its own `calico-system` namespace:

"The Calico policy-agent runs in its own `calico-system` namespace. **Create this namespace:**"